nextflow_schema.json

```diff
@@ -268,8 +268,9 @@
             "properties": {
                 "proteus_measurecol_prefix": {
                     "type": "string",
-                    "default": "LFQ intensity ",
-                    "description": "Prefix of the column names of the MaxQuant proteingroups table in which the intensity values are saved; the prefix has to be followed by the sample names that are also found in the samplesheet. Default: 'LFQ intensity '; take care to also consider trailing whitespace between prefix and samplenames."
+                    "default": "LFQ intensity_s",
```

Comment:
This seems weird and brittle- too hacky for me. People are going to misunderstand this and get it wrong. We need a different solution - why not just make the proteus module allow for an optional space after the prefix?

Reply:
I generally prefer not having my code assume that a whitespace is necessary and instead just take directly what the user inputs...but I suppose here, the flexible whitespace addition is the best solution. I just updated the proteus module with those changes